doc/source/installation.rst

```diff
@@ -87,13 +87,25 @@ Run the following commands to build the indices for the mentioned functionalitie
 | ``./scripts/build_indices.sh dbpedia_uri`` | ER (only for ELR model)  |
 +--------------------------------------------+--------------------------+
 
-3.3 Dwnload the remaining data files
-~~~~~~~~~~~~~~~~~~~~~~~~~
+3.3 Download the remaining data files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Run the following commands to download the data file needed for runing entity linking service
+Run the following commands to download the data file needed for running entity linking service
 
 +------------------------------------------------------------------------------------------+--------------+
 | Command                                                                                  | Required for |
 +==========================================================================================+==============+
 | ``wget https://gustav1.ux.uis.no/downloads/nordlys-v02/snapshot_2015_10.txt -P data/el`` | EL           |
 +------------------------------------------------------------------------------------------+--------------+
+
+3.4 Post download settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To run Nordlys smoothly the following points are to be considered:
+
+- Create a folder named "logs" and a sub-folder named "api" in Nordlys main directory, so api can create the logs file inside and starts running at ease.
```

Comment:
for the log files that'll be generated by the API.

Reply:
OK.